remotezip.py

```diff
@@ -176,7 +176,11 @@ def make_header(range_min, range_max):
     def request(url, range_header, kwargs):
         kwargs['headers'] = headers = dict(kwargs.get('headers', {}))
         headers['Range'] = range_header
-        res = requests.get(url, stream=True, **kwargs)
+        session = kwargs.pop('session', None)
```

Comment:
`kwargs` was meant to contain all parameters to be passed directly to `requests.get` (so allow some flexibility in the case the signature of that method change). In this case I think is better to make this explicit adding it to the signature of `RemoteZip` `__init__(self, url, initial_buffer_size=64*1024, session=None, **kwargs)` and storing `session` as a class instance attribute.

Reply:
Thanks for the review! Since `request()` is a static method without access to instance attributes would adding `session` as a param to `request()`'s signature be needed too?
Something like:
`def request(url, range_header, kwargs, session=None)`

Reply:
(The latest commit has what I had in mind, for reference)